modules/lb_simple_schedule/lb_simple_schedule.info.yml

```
@@ -0,0 +1,10 @@
+name: 'LB Simple Schedule'
+type: module
+description: Helper module to show "Simple Schedule" block in Layout Builder.
+core_version_requirement: ^8.8 || ^9 || ^10
+dependencies:
+  - drupal:block_content
+  - drupal:layout_builder
+  - y_pef_schedule
+  - y_lb
```

```suggestion
  - y_lb
  - drupal:plugin
```
+ add hook_update_N to enable plugin if not enabled ( optional, I guess )

---

@podarok I don't think the hook is needed since this submodule is brand new and could not have existed before. Plugin will get enabled when the module is installed.